airbyte-cdk/python/airbyte_cdk/sources/declarative/interpolation/macros.py

```diff
@@ -83,17 +84,23 @@ def max(*args):
     return builtins.max(*args)
 
 
-def day_delta(num_days: int) -> str:
+def day_delta(num_days: int, from_date: Union[str, datetime.datetime] = None) -> str:
```

Comment:
nite: the type should be `Optional[Union[str, datetime.datetime]]`

Reply:
Updated!